### ISSUE TITLE
Death Screen issues: invalid prompt, "show inventory" issue and multiple taps required to dismiss

### ISSUE THREAD

**Issue by SeymourJ** [sethhoward/iBrogueCE#6]
Upon death, this build prompts with “Press ‘i’ to view your inventory” which, again isn’t useful on a device without a hardware keyboard. Seth’s builds removed this prompt.

The “show inventory” button that appears on death in this build, is disabled by the “—more—“ prompt that appears at the same time. In Seth’s 1.75 build, you could tap “show inventory” without tapping “—more—“ first. The button probably shouldn’t be displayed when you can’t actually tap it.

You have to tap “—more—“ twice to proceed on the “you are dead” page in this build. Only once in Seth’s 1.75 build.

**Comment by SeymourJ** [comment]
As of Bob's Build on March 30 2022, the "Press i to view your inventory" prompt is gone, which is good, but the "Show Inventory" button also seems to be gone, which is not so good.

**Comment by btaylor84** [comment]
reworked the Inventory stuff at the end games for Died, Ascended, Ascended with Mastery. Now it's  a separate dialogue box, not an iPad view element.

**Comment by SeymourJ** [comment]
Bob,

Tested the latest build and this works and looks great, at least for the DIED case.   I can’t test the Ascended or Ascended with Mastery cases because I stink at Brogue  ;-)

Thanks
Seymour

**Comment by btaylor84** [comment]
yeah, I feel you. It's been a long time since I even played, let alone won.
